tests/__tests__/e2e.test.ts

```diff
@@ -1,8 +1,9 @@
 import { Agent, HttpOutboundTransporter, PollingInboundTransporter } from '../../src'
 import { getBaseConfig, waitForBasicMessage } from '../../src/__tests__/helpers'
-import logger from '../../src/__tests__/logger'
+import testLogger from '../../src/__tests__/logger'
 import { get } from '../http'
 
+const logger = testLogger
```

Comment:
Can you explain the reason for doing this one more time? I forgot :)

Reply:
Yeah. It's handy to be able to set the logger in one test to `ConsoleLogger` in this way.